core-profile/cdi-langmodel-tck/pom.xml

```diff
@@ -114,6 +112,8 @@
                                 <fileset id="jakarta-api-jars"
                                          dir="${jboss.home}${file.separator}modules/system/layers/base/jakarta/">
                                     <include name="**/annotation/api/main/*.jar"/>
+                                    <include name="**/ejb/api/main/*.jar"/>
```

Comment:
Why? Please add a comment as my "Why" will be a perpetual question since this is core-profile.

A comment for EL would be good as well.